libraries/heft-config-file/src/index.ts

```diff
@@ -27,4 +27,11 @@ export {
   type IPropertyInheritanceDefaults,
   PathResolutionMethod,
   type PropertyInheritanceCustomFunction
-} from './ConfigurationFile';
+} from './ConfigurationFileBase';
+
+export {
+  // TODO: REname this export to `ProjectConfigurationFile` in the next major version bump
+  ProjectConfigurationFile as ConfigurationFile,
```

Comment:
Is this being done mostly to ensure a smooth transition, then to flip the switch later on? It's still a 0.x package, it may be worth ripping the bandaid off now instead of having to care about it long term

Reply:
I'll export both, and then drop the other in a later release.